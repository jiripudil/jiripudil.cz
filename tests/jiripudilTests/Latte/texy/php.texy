## Title of the document

Some paragraph.

/--php
echo 'code sample';
exit(4);
\--
