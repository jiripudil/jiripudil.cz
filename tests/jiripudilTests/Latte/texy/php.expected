<h2>Title of the document</h2>

<p>Some paragraph.</p>

<pre><code><span class="php-keyword1">echo</span> <span
class="php-quote">'code sample'</span>;
<span
class="php-keyword1">exit</span>(<span class="php-num">4</span>);</code></pre>
